### ISSUE TITLE
Network printer fails because of ipp backend permissions?

### ISSUE THREAD

**Issue by Keruspe** [apple/cups#5715]
Hello,

I have cups setup with avahi and cups-browsed is started to discover network printers.
When I try to use the printer, I get a message telling me that `/usr/libexec/cups/backend/ipp` failed. I notice it has `0700` permissions. Making it `0755` "fixes" the problem.
I guess that's not the right fix, as there are good security reasons for it to be `0700` but I'm not sure how to find out what's going on and why it's needed here to work?

**Comment by Keruspe** [comment]
error logs are:

```
E [06/Jan/2020:10:45:43 +0100] [Job 26] File \'\' not found
E [06/Jan/2020:10:45:43 +0100] [Job 26] Could not start IPP Backend (/usr//libexec/cups/backend/ipp): 13 Permission denied
```

**Comment by zdohnal** [comment]
Hi,
cups-filters upstream already solved the problem in 1.26.1. Basically, implicitclass backend now has permissions 700 by default and is run as root, so IPP backend, which implicitclass starts, is run as root too.

**Comment by Keruspe** [comment]
Oh, I'm running 1.26.0 it seems, thanks for the pointer !
